crud/common/utils.lua

```diff
@@ -488,4 +493,35 @@ function utils.flatten_obj_reload(space_name, obj)
     return schema.wrap_func_reload(flatten_obj, space_name, obj)
 end
 
+function utils.format_jsonpath(operand, field_names, field_ids)
+    local ind = operand:find("]")
+    if ind == nil or keydef == nil then -- not jsonpath OR tarantool < 2
+        return {
+            operand = operand,
+        }
+    end
+
+    local top_field = operand:sub(0, ind)
+    local ok, new_keydef = pcall(keydef.new, {{type = 'string', fieldno = 1, path = top_field}})
```

Comment:
`keydef` is a module, but `new_keydef` is a keydef object.

Use `key_def_lib = require('tuple.keydef')` like in `crud/select/merger.lua`.